code/modules/mob/living/simple_animal/abnormality/_auxiliary_modes/community/ego_weapons.dm

```diff
@@ -248,7 +248,7 @@
 	if(!can_spin)
 		to_chat(user,span_warning("You attacked too recently."))
 		return
-	if(do_after(user, 12, src))
+	if(do_after(user, 8, src))
```

Comment:
Why did you reduce the cooldown on this?

Reply:
Some testing happened to reveal that it needed a buff.